lang/update_pot.sh

```diff
@@ -32,6 +32,7 @@ xgettext --default-domain="cataclysm-dda" \
          --keyword="gettext_noop" \
```

Comment:
Because `gettext_noop` looks confusing for someone unfamiliar with gettext library, while `translate_marker` expresses its intention to mark strings for translation well.